README.md

```diff
@@ -83,6 +83,9 @@ It is:
 * a Kubernetes controller capable of centrally managing both VxLAN and VLAN interfaces of all Kubernetes hosts
 * another Kubernetes controller extending Kubernetes' Service-based service discovery concept to work over all network interfaces of a Pod
 * a standard Kubernetes Validating and Mutating Webhook responsible for making you adhere to the schemas, and also automating network resource management for tenant users in a production-grade environment
+### Install an Akraino REC and get DANM for free!
```

Comment:
the table of contents also needs to be refreshed because of this, as the indentation and the placement of the paragraph changed